networkx/algorithms/Maximum Priority Matching/maximum_priority_matching.py

```diff
@@ -0,0 +1,2009 @@
+import doctest
```

Comment:
Add file heading, with explanations, article name and authors, and programmer names.